lerna.json

```diff
@@ -0,0 +1,12 @@
+{
+  "packages": ["packages/*"],
```

Comment:
If we are providing react specific quality control. Lets structure our packages in a slightly different manner.

Proposed structure:

packages
 -> react
      -> javascript
           -> eslint-config
           -> stylelint-config

Lerna.json

`
{
  "packages": ["packages/react/javascript/eslint-config", "packages/react/javascript/styleline-config"],
}
`

OR we can change the repo name to react-quality-control or react-standards :)

Reply:
@ankutr that was not the ask and intent to create react specific. user can also turned off any rules or update rules if required. Why do you think its react specific only?

Reply:
Yeah. Thats why I mentioned an OR condition. :) If the name of the repo is generic, the code and the relevant structure should support additions

Reply:
@ankurkakroo2  You wil be discussing this approach with vinod and confirm.

Reply:
@ankurkakroo2 I like your idea, we could use the same setup for our future Vue work or any thing else in future. 

My thought was to create that differentiation in package name itself, like @xt-pagesource/react-next-eslint